Ghost_in_the_Shell/client_folder_react_app/ethisim/src/components/scenario_components/Action.js

```diff
@@ -1,16 +1,88 @@
 import React from 'react';
-import Typography from '@material-ui/core/Typography';
+import {
+  Button,
+  TextField,
+  Typography,
+  Container,
+  Box,
+} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
 import Introduction from '../Introduction';
 import Title from '../Title';
+import Copyright from '../components/Copyright';
+
+const useStyles = makeStyles((theme) => ({
+  container: {
+    marginTop: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  form: {
+    marginTop: theme.spacing(1),
+    width: '100%',
+  },
+  submit: {
+    marginTop: theme.spacing(2),
+    backgroundColor: theme.palette.primary.main,
+    color: 'white',
+  },
+  copyright: {
+    margin: theme.spacing(2),
+    opacity: 0.5,
+  }
+}));
 
 export default function FinalAction() {
+  const classes = useStyles();
+  
   return (
-    <div>
+    <Container component="main">
       <Typography align="center" variant="h2">
         Action Component
       </Typography>
       <Title />
       <Introduction />
-    </div>
+      <div className={classes.container}>
+        <form className={classes.form}>
+          <Typography align="center" variant="h6">
+            Option 1
+          </Typography>
+          <TextField
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            id="option 1"
+            label="Input Option 1 Text"
+            name="option 1"
+            autoFocus
+          />
```

Comment:
Get rid of autoFocus, page scrolls down to option 1

Reply:
Corrected